.github/workflows/include-deploy-release.yml

```diff
@@ -18,33 +18,33 @@ jobs:
 
     steps:
       - name: git checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           submodules: "recursive"
 
       - name: Setup Java 11
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
           distribution: ${{ vars.VERSION_JAVA_DISTRIBUTION  }}
           java-version: ${{ vars.VERSION_JAVA }}
 
       - name: Setup Gradle and task/dependency caching
-        uses: gradle/gradle-build-action@v2
+        uses: gradle/actions/setup-gradle@v3
         with:
           cache-read-only: false
 
       - name: Setup cmake
-        uses: jwlawson/actions-setup-cmake@v1.13
+        uses: jwlawson/actions-setup-cmake@v2
         with:
           cmake-version: ${{ vars.VERSION_CMAKE }}
 
       - name: Setup ninja
-        uses: cmelchior/setup-ninja@master
+        uses: clementetb/setup-ninja@master
```

Comment:
is this still based on https://github.com/ashutoshvarma/setup-ninja ?

Reply:
It is based in cmelchior/setup-ninja that it is based on ashutoshvarma's